account_usability/account.py

```diff
@@ -170,6 +174,23 @@ def py3o_lines_layout(self):
         # ]
         return res
 
+    def _compute_sales_dates(self):
+        """ French law requires to set sale order dates into invoice
+            returned string: "sale1 (date1), sale2 (date2) ..."
+        """
+        for inv in self:
+            sale_ids = [x.sale_line_ids.mapped('order_id')[0].id
+                        for x in inv.invoice_line_ids]
+            sales = self.env["sale.order"].browse(set(sale_ids))
+            lang = inv.partner_id.commercial_partner_id.lang
+            date_format = self.env["res.lang"]._lang_get(
+                lang or "").date_format
+            dates = ["%s%s" % (
+                     x.name, x.confirmation_date and
+                     " (%s)" % x.confirmation_date.strftime(date_format) or "")
```

Comment:
illsibile

Reply:
you're right but I don't know to format in better way

Reply:
done but not sure it's better